### PR TITLE
Feat/indikator 5 6

### PR DESCRIPTION
**Pull request by AwwalaK**
**Text Finding for indicator 5 & 6**

- checked for keywords in lvl 2, lvl 3, and lvl 4
- Tested on:

1. Dokumen PhakPhakBarat

> Indikator 5
Screen Shot 2023-03-17 at 09 57 24

> Indikator 6
Screen Shot 2023-03-17 at 10 00 16

3. Dokumen Batubara

> Indikator 5
Screen Shot 2023-03-17 at 09 58 11

> Indikator 6
Screen Shot 2023-03-17 at 09 59 48

5. Dokumen Lamongan Lama

> Indikator 5
Screen Shot 2023-03-17 at 09 58 36

> Indikator 6
Screen Shot 2023-03-17 at 10 01 30

### CODE REVIEW

File: textfinding_indikator6.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    lvl2 = convert_keywords([
+        'organisasi perangkat daerah', 
+        'opd', 
+        'unit kerja', 
+        'pemerintah daerah', 
+        'perangkat daerah'])
+    res2 = txtreader(filename, 2, lvl2)
+
+    if (not res2):
+        return clean_text(list_final)
```
Comment (akramab) [inline textfinding_indikator6.py:60]: same comment as https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7#discussion_r1142027756

File: textfinding_indikator6.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            if (re.search(reg1, line.lower(), re.IGNORECASE) or re.search(reg2, line.lower(), re.IGNORECASE)):
+                for key in keyword:
+                    if re.search(key, line.lower(), re.IGNORECASE):
+                        result.append([idx, line])
+
+        else:
+            if (re.search(reg1, line.lower(), re.IGNORECASE) or re.search(reg2, line.lower(), re.IGNORECASE)):
+                res = [ele for ele in keyword if (ele in line)]
+                if (res):
+                    result.append([idx, line])
```
Comment (akramab) [inline textfinding_indikator6.py:35]: same comment as https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7#discussion_r1142026036

File: .DS_Store
Comment (akramab) [inline .DS_Store:1]: Also please don't forget to unstage this file from the commit ^_^)b

**Review by akramab — APPROVED** [review]

File: textfinding_indikator5.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    lvl2 = convert_keywords([
+        'organisasi perangkat daerah', 
+        'opd', 
+        'unit kerja', 
+        'pemerintah daerah', 
+        'perangkat daerah'])
+    res2 = txtreader(filename, 2, lvl2)
+
+    if (not res2):
+        return ''
```
Comment (akramab) [inline textfinding_indikator5.py:53]: Check this comment here https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7/files/9281deef51004de3bc32dd4947f2b33504c09d63#r1141619220

This should return the lines found from lvl1 (the values in `res1` variable)

File: textfinding_indikator6.py [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    lvl2 = convert_keywords([
+        'organisasi perangkat daerah', 
+        'opd', 
+        'unit kerja', 
+        'pemerintah daerah', 
+        'perangkat daerah'])
+    res2 = txtreader(filename, 2, lvl2)
+
+    if (not res2):
+        return ''
```
Comment (AlifahRBasyasya) [inline textfinding_indikator6.py:53]: Same as [#7 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/7/files#r1142390863)

**Review by akramab — APPROVED** [review]
LGTM, nicely done

**Review by afifahfq — APPROVED** [review]

**Pull request merged**